cmd/argo/commands/root.go

```diff
@@ -132,3 +137,12 @@ If your server is behind an ingress with a path (you'll be running "argo server
 
 	return command
 }
+
+// setGLogLevel set the glog level for the k8s go-client
+// this is taken from argoproj/pkg but uses v2 of klog here
+// to be compatible with k8s clients v0.19.x and above
+func setGLogLevel(glogLevel int) {
```

Comment:
this needs to be fixed in three places in total:

* here
* cmd/argoexec/commands/root.go
* cmd/workflow-controller/main.go

Reply:
Sounds good - I moved the function to the `util/cmd` package and then invoked it in each of the files you listed